packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3576,42 +3576,3 @@ class InputDecorationTheme extends Diagnosticable {
     properties.add(DiagnosticsProperty<bool>('alignLabelWithHint', alignLabelWithHint, defaultValue: defaultTheme.alignLabelWithHint));
   }
 }
-
-/// The vertical alignment of text within an input.
-///
-/// A single [y] value that can range from -1.0 to 1.0. -1.0 aligns to the top
-/// of the input so that the top of the first line of text fits within the input
-/// and its padding. 0.0 aligns to the center of the input. 1.0 aligns so that
-/// the bottom of the last line of text aligns with the bottom interior edge of
-/// the input.
-///
-/// See also:
-///
-///  * [TextField.textAlignVertical], which is passed on to the [InputDecorator].
-///  * [InputDecorator.textAlignVertical], which defines the alignment of
-///    prefix, input, and suffix, within the [InputDecorator].
-class TextAlignVertical {
```

Comment:
I just moved this to alignment.dart so that it's accessible by both Cupertino and Material.